etc/kayobe/ansible/octavia-amphora-image-register.yml

```
@@ -13,21 +11,57 @@
         lookup('env', 'OS_USERNAME') != 'octavia' or
         lookup('env', 'OS_PROJECT_NAME') != 'service'
 
+- name: Download Amphora image from Ark
+  ansible.builtin.import_playbook: pulp-amphora-image-download.yml
+  when: download_from_ark | default(true) | bool
```

```suggestion
  when: download_amphora_from_ark | default(true) | bool
```
This will make it clear what the variable does if we want to override it in globals for a customer.

---

Also, I think it would be good to define this in `stackhpc.yml`  as true, instead of putting the default statement here.